app/arenas/camelot.py

```diff
@@ -0,0 +1,27 @@
+from app.knights.knight import Knight
+
+
+class Camelot:
+
+    battles_results = {}
+
+    def __init__(self) -> None:
+        pass
+
+    @classmethod
+    def knight_battle(cls, knight_one: Knight, knight_two: Knight) -> dict:
+        knight_one.hp -= knight_two.power - knight_one.protection
+        knight_two.hp -= knight_one.power - knight_two.protection
+
+        if knight_one.hp < 0:
+            print(f"{knight_one.name} fall in battle!")
+            knight_one.hp = 0
+
+        if knight_two.hp < 0:
+            print(f"{knight_two.name} fall in battle!")
+            knight_two.hp = 0
```

Comment:
u can use `max(knight_one.hp, 0)`

Reply:
Thanks you for advice. Seen this construction used many times in morning tasks, but when it's time to use it - I always forget about it.